tool/stats_lib.dart

```diff
@@ -15,18 +15,20 @@ import '../test/util.dart';
 
 // Locate the "tool" directory. Use mirrors so that this works with the test
 // package, which loads this suite into an isolate.
-String get toolDir =>
-    p.dirname((reflect(loadCommonMarkSections) as ClosureMirror)
-        .function
-        .location!
-        .sourceUri
-        .path);
+String get toolDir => p.dirname(
```

Comment:
I think this should be reverted.

But this change I think indicates a smell; this arrow-function should probably be a function body with a local variable, like a variable for the `reflect` result...